creator-node/src/middlewares.js

```diff
@@ -141,15 +141,32 @@ async function ensureStorageMiddleware (req, res, next) {
  * @dev - TODO move this out of middlewares to Services layer
  */
 async function triggerSecondarySyncs (req) {
-  if (config.get('isUserMetadataNode') || config.get('snapbackDevModeEnabled')) return
+  const { snapbackSM } = serviceRegistry
```

Comment:
every change in this file is just code cleanup, except for line 165 (the `enqueueManualSync()` call